homeassistant/components/overkiz/number.py

```diff
@@ -97,6 +97,28 @@ async def _async_set_native_value_boost_mode_duration(
         max_value_state_name=OverkizState.CORE_MAXIMAL_SHOWER_MANUAL_MODE,
         entity_category=EntityCategory.CONFIG,
     ),
+    OverkizNumberDescription(
+        key=OverkizState.CORE_TARGET_DWH_TEMPERATURE,
+        name="Target Temperature",
+        icon="mdi:thermometer",
+        command="setTargetDHWTemperature",
```

Comment:
Could you do a PR to add this value? https://github.com/iMicknl/python-overkiz-api/blob/main/pyoverkiz/enums/command.py. We only accept enums as values at the moment.

Reply:
Okay, I will.

Reply:
Done https://github.com/iMicknl/python-overkiz-api/pull/1187

Reply:
Great! After pyoverkiz is updated to v1.13.9 in a follow-up PR, you can rebase this PR and add the constant. Thanks.

Reply:
@ALERTua merged, you can rebase now :-)

Reply:
Sorry, I haven't used git rebase, so I did it the way I knew how - merged.